app/src/main/res/layout/activity_main.xml

```diff
@@ -159,7 +159,7 @@
             android:layout_width="match_parent"
             android:layout_height="match_parent"
             android:layout_weight="1"
-            android:text="*"/>
```

Comment:
Good catch!

Reply:
hey nice one didn't see that one before !